notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,411 @@
+    "            else:\n",
+    "                # Encode numpy array to jpg.\n",
+    "                _, encoded_img = cv2.imencode(\".jpg\", result_image, params=[cv2.IMWRITE_JPEG_QUALITY, 90])\n",
+    "                # Create an IPython image.\n",
+    "                i = display.Image(data=encoded_img)\n",
+    "                # Display the image in this notebook.\n",
+    "                display.clear_output(wait=True)\n",
+    "                display.display(i)\n",
+    "    # ctrl-c\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    # any different error\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "    finally:\n",
+    "        if player is not None:\n",
+    "            # Stop capturing.\n",
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Action Recognition Using a Webcam\n",
+    "\n",
+    "Now, try to see yourself in your webcam. By default, the primary webcam is set with `source=0`. If you have multiple webcams, each one will be assigned a consecutive number starting at 0. Set `flip=True` when using a front-facing camera. Some web browsers, especially Mozilla Firefox, may cause flickering. If you experience flickering, set `use_popup=True`.\n",
+    "\n",
+    "> **NOTE**: To use a webcam, you must run this Jupyter notebook on a computer with a webcam. If you run on a server, the webcam will not work. However, you can still do inference on a video file in the final step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_style_transfer(source=2, flip=True, use_popup=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Action Recognition on a Video File\n",
+    "\n",
+    "Find out how the model works in a video file. [Any format supported](https://docs.opencv.org/4.5.1/dd/d43/tutorial_py_video_display.html) by OpenCV will work. You can press the stop button anytime while the video file is running, and it will activate the webcam for the next step.\n",
+    "\n",
+    "> **NOTE**: Sometimes, the video can be cut off if there are corrupted frames. In that case, you can convert it. If you experience any problems with your video, use the [HandBrake](https://handbrake.fr/) and select the MPEG format."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "test_replace": {
+     "use_popup=False": "use_popup=False, skip_first_frames=280"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "video_file = \"https://github.com/intel-iot-devkit/sample-videos/blob/b8e3425998213e0b4957c20f0ed5f83411f7a802/person-bicycle-car-detection.mp4?raw=true\"\n",
```

Comment:
I think we could find a more exciting video than this one. Please take a look here: https://github.com/intel-iot-devkit/sample-videos

Reply:
Sure. I will pick an interesting one.